pkg/monitortests/network/disruptionserviceloadbalancer/monitortest.go

```diff
@@ -344,3 +354,29 @@ func httpGetNoConnectionPoolTimeout(url string, timeout time.Duration) (*http.Re
 
 	return client.Get(url)
 }
+
+// Uses the first node in the cluster to verify the LoadBalancer host is active before returning
+func checkHostnameReady(ctx context.Context, tcpService *corev1.Service, nodeTgt string) error {
+	oc := exutil.NewCLIForMonitorTest(tcpService.GetObjectMeta().GetNamespace())
+
+	var (
+		stdOut string
+		err    error
+	)
+
+	wait.PollUntilContextTimeout(ctx, 15*time.Second, 60*time.Minute, true, func(ctx context.Context) (bool, error) {
```

Comment:
Just double checking, this is a conscious choice not to capture the err returned from PollUntilContextTime ( could be context closed or timeout, etc.) and only return the last err, if any, returned from oc.AsAdmin call?

Reply:
Yes it was intentional to not capture the err

Reply:
I have to admit I'm curious.  So this allows for a path where PollUntilContextTimeout returns an error but checkHostNameReady does not and that will allow StartCollection to proceed and presumably error out down the line?

Reply:
Yes it will error out here right after the checkHostNameReady function call since the service LoadBalancer is not verified
```      
fmt.Fprintf(os.Stderr, "hitting pods through the service's LoadBalancer\n")
timeout := 10 * time.Minute
```
The test will time out

Reply:
Ahh, thanks for the explanation!